edgecast/rtld/lookups/lookups_client.go

```diff
@@ -8,19 +8,20 @@ package lookups
 import (
 	"fmt"
 
-	"github.com/EdgeCast/ec-sdk-go/edgecast/client"
+	"github.com/EdgeCast/ec-sdk-go/edgecast/internal/ecclient"
 )
 
 // New creates a new lookups API client.
-func New(c client.Client) ClientService {
-	return &Client{Client: c}
+func New(c ecclient.APIClient, cc ecclient.ClientConfig) ClientService {
+	return &Client{c, cc}
 }
 
 /*
 Client for lookups API
 */
 type Client struct {
```

Comment:
Is there any way we can call these "Service" to avoid confusion with the API Client?